pyproject.toml

```
@@ -0,0 +1,54 @@
+[build-system]
+requires = ["flit_core >=3.2,<4"]
+build-backend = "flit_core.buildapi"
+
+[project]
+# Check https://flit.readthedocs.io/en/latest/pyproject_toml.html for all available sections
+name = "ansys-dpf-core"
+version = "0.7.dev0"
+description = "Data Processing Framework - Python Core "
+readme = "README.md"
+requires-python = ">=3.7"
```

```suggestion
requires-python = ">=3.7, <3.11"
```
Not sure this is required but it might be?